docs/services/azure-services-guide.md

```diff
@@ -0,0 +1,625 @@
+balancer `Service`. We will point the deployment to the `wordpressmysql` secret
+that we specified in our claim above for the Wordpress container environment
+variables. It should have been populated with our MySQL connection details after
+the claim became `Bound`.
+
+* Check to make sure `wordpressmysql` exists and is populated:
+
+*Command*
+```bash
+kubectl describe secret wordpressmysql -n app-project1-dev
+```
+
+*Output*
+```bash
+Name:         wordpressmysql
+Namespace:    app-project1-dev
+Labels:       <none>
+Annotations:  <none>
+
+Type:  Opaque
+
+Data
+====
+endpoint:  75 bytes
+password:  27 bytes
+username:  58 bytes
+```
+
+* Define the `Deployment` and `Service` in `wordpress-app.yaml`:
+```yaml
+cat > wordpress-app.yaml <<EOF
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  namespace: app-project1-dev
+  name: wordpress
+  labels:
+    app: wordpress
+spec:
+  selector:
+    matchLabels:
+      app: wordpress
+  template:
+    metadata:
+      labels:
+        app: wordpress
+    spec:
+      containers:
+        - name: wordpress
+          image: wordpress:4.6.1-apache
+          env:
+            - name: WORDPRESS_DB_HOST
+              valueFrom:
+                secretKeyRef:
+                  name: wordpressmysql
+                  key: endpoint
+            - name: WORDPRESS_DB_USER
+              valueFrom:
+                secretKeyRef:
+                  name: wordpressmysql
+                  key: username
+            - name: WORDPRESS_DB_PASSWORD
+              valueFrom:
+                secretKeyRef:
+                  name: wordpressmysql
+                  key: password
+          ports:
+            - containerPort: 80
+              name: wordpress
+---
+apiVersion: v1
+kind: Service
+metadata:
+  namespace: app-project1-dev
+  name: wordpress
+  labels:
+    app: wordpress
+spec:
+  ports:
+    - port: 80
+  selector:
+    app: wordpress
+  type: LoadBalancer
+EOF
+```
+
+* Create the `Deployment` and `Service`:
+```bash
+kubectl create -f wordpress-app.yaml
+```
+
+* You can verify creation with the following command and output:
+
+*Command*
+```bash
+kubectl get -f wordpress-deployment.yaml 
+```
+*Output*
+```bash
+NAME                        READY   UP-TO-DATE   AVAILABLE   AGE
+deployment.apps/wordpress   1/1     1            1           11m
+
+NAME                TYPE           CLUSTER-IP    EXTERNAL-IP   PORT(S)        AGE
+service/wordpress   LoadBalancer   10.0.128.30   52.168.69.6   80:32587/TCP   11m
+```
+
+If the `EXTERNAL-IP` field of the `LoadBalancer` is `<pending>`, wait until it
+becomes available, then navigate to the address. You should see the following:
+
+![alt azure wordpress](azure-wordpress.png)
+
+## Uninstall
+
+### Wordpress
+All Wordpress components that we installed using Helm can be deleted with one
+command:
+
+```bash
+kubectl delete -f wordpress-app.yaml
+```
+
+### Crossplane
+To delete all created resources, but leave Crossplane and the Azure stack
```

Comment:
It is a bit confusing that this section is called `Crossplane` but seems to indicate that we are leaving Crossplane alone. I might suggest calling the section something more like  `Crossplane Configuration` or `Crossplane-side Configuration`. I dunno